lua/neosolarized.lua

```diff
@@ -213,6 +213,31 @@ function M.setup(opts)
     Group.new('CmpItemMenu', groups.NormalNC, colors.bg, styles.none)
     -- Group.new('CmpItemAbbr', colors.base0, colors.bg, styles.none)
     -- Group.new('CmpItemAbbrMatch', colors.base0, colors.bg, styles.none)
+    Group.new("CmpItemKindText", colors.base3, colors.none, styles.none)
+    Group.new("CmpItemKindMethod", colors.green, colors.none, styles.none)
+    Group.new("CmpItemKindFunction", colors.blue, colors.none, styles.none)
+    Group.new("CmpItemKindConstructor", colors.orange, colors.none, styles.none)
+    Group.new("CmpItemKindField", colors.yellow, colors.none, styles.none)
+    Group.new("CmpitemkindVariable", colors.orange, colors.none, styles.none)
```

Comment:
should this be CmpItemKindVariable? Snake-case out of whack in a few others here too, is that intentional?

Reply:
Fixed. Sorry, my bad.